### ISSUE TITLE
Model hyperparams are inconsistent

### ISSUE THREAD

**Issue by ckswjd99** [omihub777/MLP-Mixer-CIFAR#2]
Hello, I trained MLP Mixer with your codes and it works very well.

Meanwhile, I found that the model trained by `main.py` with the sample command in `README.md` has about 1.1M params, rather than 0.67M as written in readme.

After reading some codes, I found that the `hidden_s` and `hidden_c` values are sometimes reversed.

For example, in `mlp_mixer.py`, hidden_s=512 and hidden_c=64, and it makes 0.67M model.

```
if __name__ == '__main__':
    net = MLPMixer(
        in_channels=3,
        img_size=32, 
        patch_size=4, 
        hidden_size=128, 
        hidden_s=512, 
        hidden_c=64, 
        num_layers=8, 
        num_classes=10, 
        drop_p=0.,
        off_act=False,
        is_cls_token=True
        )
    torchsummary.summary(net, (3,32,32))

```

However in `main.py`, hidden-c=512 and hidden-s=64, and it makes 1.1M model.

```
parser.add_argument('--hidden-c', type=int, default=512)
parser.add_argument('--hidden-s', type=int, default=64)
```

When I trained 1.1M model, I could reach 90.06% validation accuracy.

When I trained 0.67M model, I could reach 84.81% validation accuracy.

I just wanted to report my observation, and if I misunderstood something, please let me know.

Thanks for sharing your nice codes!
